Remotion/Core/Collections.Caching.UnitTests/Core.Collections.Caching.UnitTests.csproj

```diff
@@ -83,6 +100,10 @@
       <Project>{b719a305-0493-4c72-88fe-9a2fc6964578}</Project>
       <Name>Development.Core</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\Development\Moq\Development.Moq.csproj">
+      <Project>{f49b0545-2ad6-401e-8db9-f11a3e01c07b}</Project>
+      <Name>Development.Moq</Name>
+    </ProjectReference>
     <ProjectReference Include="..\..\Development\RhinoMocks\Development.RhinoMocks.csproj">
```

Comment:
Remvoe RhinoMocks. Actually, the RhinoMocks dependency itself is also still there. Please check other csproj and packages.config files, too.